Documentation/Signoffs/CANCapturing.md

```diff
@@ -0,0 +1,84 @@
+CAN Bus Capturing
+---------------------------
+### Function of the Subsystem
+This subsystem must capture the data on the CAN (Controller Area Network) bus 
+and store it. This system is not responsible for interpreting the information
+that is captured.
+
+### Specifications & Constraints
+- Must capture information from CAN bus at no less than 10 Hz
+- Must store CAN bus data neatly and in a readable manner on the PC
+- Shall not interfere with the functionality with any other subsystem
+- Shall receive power from the PC
+
+### Wiring Schematic
+
+![image](https://user-images.githubusercontent.com/100802413/215526784-682a9f8f-5de8-495e-bd05-458638799d19.png)
+
+###### _Figure 1: Arduino Leonardo CAN BUS Board Connection to OBD II Port via DB9 connector_
+
+### Analysis
+##### 1. Microcontroller Selection
```

Comment:
I'm unclear on the purpose of this section? To show that the pins will work, add the pinout information with citation to your schematic. Remove any extraneous information.